### PR TITLE
pnpm: add German translation

### PR DESCRIPTION
**Pull request by xRealNeon**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> Welcome to tldr

File: pages.de/common/pnpm.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`pnpm init`
+
+- Installiere alle in der `package.json` Datei gelisteten Abhängigkeiten:
+
+`pnpm install`
+
+- Installiere eine spezifische Version eines Packages und füge es automatisch der `package.json` Datei hinzu:
+
+`pnpm add {{module_name}}@{{version}}`
```
Comment (pixelcmtd) [inline pages.de/common/pnpm.md:17]:
```suggestion
`pnpm add {{modulname}}@{{version}}`
```

File: pages.de/common/pnpm.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Installiere ein Package und füge es als Entwicklungs-Abhängigkeit der `package.json` Datei hinzu:
+
+`pnpm add -D {{module_name}}`
```
Comment (pixelcmtd) [inline pages.de/common/pnpm.md:21]:
```suggestion
`pnpm add -D {{modulname}}`
```

File: pages.de/common/pnpm.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Installiere ein Package global:
+
+`pnpm add -g {{module_name}}`
```
Comment (pixelcmtd) [inline pages.de/common/pnpm.md:25]:
```suggestion
`pnpm add -g {{modulname}}`
```

File: pages.de/common/pnpm.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Deinstalliere ein Package und entferne es automatisch aus der `package.json` Datei:
+
+`pnpm remove {{module_name}}`
```
Comment (pixelcmtd) [inline pages.de/common/pnpm.md:29]:
```suggestion
`pnpm remove {{modulname}}`
```

File: pages.de/common/pnpm.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# pnpm
+
+> Schneller, speicherplatzsparender Paketmanager für Node.js.
+> Ein Kommandozeilenwerkzeug für die Verwaltung von JavaScript und Node.js Paketen (Packages).
+> Weitere Informationen: <https://pnpm.io>.
+
+- Erstelle eine `package.json` Datei interaktiv::
```
Comment (pixelcmtd) [inline pages.de/common/pnpm.md:7]:
```suggestion
- Erstelle eine `package.json` Datei interaktiv:
```

File: pages.de/common/pnpm.md (hunk 1 above)
Comment (xRealNeon) [reply pages.de/common/pnpm.md:17]: Maybe with underscore "modul_name"?

File: pages.de/common/pnpm.md (hunk 2 above)
Comment (xRealNeon) [reply pages.de/common/pnpm.md:21]: Maybe with underscore "modul_name"?

File: pages.de/common/pnpm.md (hunk 3 above)
Comment (xRealNeon) [reply pages.de/common/pnpm.md:25]: Maybe with underscore "modul_name"?

File: pages.de/common/pnpm.md (hunk 4 above)
Comment (xRealNeon) [reply pages.de/common/pnpm.md:29]: Maybe with underscore "modul_name"?

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**